### PR TITLE
Update versions and workflow

### PR DESCRIPTION
**Pull request by piraka9011**
Signed-off-by: Anas Abou Allaban <allabana@amazon.com>

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -2,6 +2,9 @@ name: Lint ROS2 package
 on:
   # Run this action whenever a pull request is made
   pull_request:
+  push:
+    branches:
+      - master
```
Comment (dabonnie) [inline .github/workflows/lint.yml]: Why add the check for pushing to master now?

Comment (piraka9011) [reply .github/workflows/lint.yml]: It wasn't there before when it should have been.
If it's not there, the Workflow badge doesn't accurately display the build status.

Comment (zmichaels11) [reply .github/workflows/lint.yml]: In future PRs, I think this should be a separate 1-liner PR.

Comment (emersonknapp) [reply .github/workflows/lint.yml]: Should it be that in this PR?

**Pull request merged**